public/random/Dare.txt

```diff
@@ -1,135 +1,216 @@
 For a guy, put on makeup. For a girl, wash off your make up.
-
 Dance with no music for 1 minute.
-
 Try to drink a glass water while standing on your hands.
-
 Let the group pose you in an embarrassing position and take a picture.
-
 Curse like sailor for 20 seconds straight.
-
 Do four cartwheels in row.
-
 Play a song on by slapping your butt cheeks till someone guesses the song.
-
 Give someone your phone and let them send one text to anyone in your contacts.
-
 Depict a human life through interpretive dance.
-
 Lick the floor.
-
-Drink a small cup of a concoction that the group makes. (Don�t make anything that might cause bodily harm. No visits to the emergency room.).
-
+Drink a small cup of a concoction that the group makes. (Don�t make anything that might cause bodily harm. No visits to the emergency room.).
```

Comment:
To get rid of the � symbols, change the text from within VS Code, rather than from a text editor.